block-storage/src/main/scala/coop/rchain/blockstorage/util/io/package.scala

```diff
@@ -64,7 +64,7 @@ package object io {
     handleIo(Files.isRegularFile(path), UnexpectedIOError.apply)
 
   def makeDirectory[F[_]: Sync: RaiseIOError](dirPath: Path): F[Boolean] =
-    handleIo(dirPath.toFile.mkdir(), UnexpectedIOError.apply)
+    handleIo(dirPath.toFile.mkdirs(), UnexpectedIOError.apply)
```

Comment:
`File.mkdirs()` doesn't fail. It only returns false. Use https://docs.oracle.com/javase/10/docs/api/java/nio/file/Files.html#createDirectories(java.nio.file.Path,java.nio.file.attribute.FileAttribute...)

Reply:
I'm wondering. `mkdirs` creates folder ", along with all necessary parent directories". Soo... when could it potentially fail? I guess read only file system etc?

Reply:
http://man7.org/linux/man-pages/man2/mkdir.2.html

Reply:
It can fail because of quite a few reasons I believe: when one of the parents already exists and is not a directory (i.e. a file), having not enough rights to create directories/files, not enough disk space, read-only fs etc

Reply:
👍